pkg/metrics/metrics.go

```diff
@@ -0,0 +1,63 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const (
+	separator = "_"
+
+	subsystem = "stonesoup_sprayproxy"
+
+	github                    = "github"
+	githubRequestsName        = subsystem + separator + github + separator + "requests_total"
```

Comment:
Similarly, this project isn't specific to GitHub. I'm going to propose that we add "inbound" as an adjective to distinguish this from potential outbound metrics.

```suggestion
	inboundRequestsName        = subsystem + separator + "inbound_requests_total"
```

May also want to consider adding http, since that is pretty common: `http_inbound_requests_total`

Reply:
part of next push